fail2ban/server/database.py

```diff
@@ -202,6 +217,9 @@ def __init__(self, filename, purgeAge=24*60*60):
 						Fail2BanDb.__version__, version, newversion)
 					raise RuntimeError('Failed to fully update')
 		finally:
+			# pypy: set journal mode after possible upgrade db:
+			if pypy:
+				cur.execute("PRAGMA journal_mode = MEMORY")
```

Comment:
just expressing my excitement that we do still have pypy3 in travis and thus this line is test covered!
BTW -- are you guys using codecov extension for firefox?  whenever it works -- it is wonderful.  I can see what is covered or not right here in the PR on github : http://www.onerussian.com/tmp/gkrellShoot_05-20-16_094317.png   and here all of the thick greens are for the new code which is covered.